code/modules/projectiles/guns/energy/nuclear.dm

```diff
@@ -289,3 +289,28 @@
 		)
 
 	has_item_ratio = FALSE
+
+/obj/item/gun/energy/psipistol
+	name = "psionic pistol"
+	desc = "A pistol issued to Federation diplomats that utilises psionics. It's made from a lightweight alloy."
```

Comment:
I would suggest to make this generic and not fed rep exclusive, so it could be used for events, or third party ships, away sites, ghost roles, etc

Reply:
done